packages/core/test/test-util/test-services/openapi/test-service/schema/index.ts

```diff
@@ -3,6 +3,7 @@
  *
  * This is a generated file powered by the SAP Cloud SDK for JavaScript.
  */
+export * from './schema-123456';
```

Comment:
this file is missing.

Reply:
Interesting, the `check:test-service` in the pipeline passed.

Reply:
As discussed, the `check:test-service` should fail as well as the compilation, while the pipeline was green.
Please create a follow-up ticket for the false-green pipeline.

Reply:
added a follow up ticket.